src/main/java/AttendanceTracker/DisplayCourses.java

```diff
@@ -0,0 +1,121 @@
+package AttendanceTracker;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.sql.Array;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+
+public class DisplayCourses {
+
+        // TODO: Create funciton that calls the year and semester function to create word ie (Spring 2023)
+        public static String buildSemesterYear(Map<String, List<String>> Map) {
+            System.out.println("\nTerms options: ");
+            for (Map.Entry<String, List<String>> entry : Map.entrySet()){
+                String key = entry.getKey();
+                System.out.println(key);
+            }
+            System.out.println("");
+            YearOptions userChooseYearStyle = new YearOptions();
+            int year = userChooseYearStyle.userChooseYearStyle();
```

Comment:
This won't work because the void functions are tightly coupled with the year options, I think Tia's code is supposed to fix this.